charts/mediawiki/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: "1.37"
 description: A Helm wbstack flavoured MediaWiki
 name: mediawiki
-version: 0.7.0
+version: 0.7.1
```

Comment:
naturally we are still claiming to be in the "nothing counts" part of sem-ver e.g. <=1.0.0 but this is probably a breaking change